src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/DELETE_Resources_LinkedHearingGrp_UnitTests.java

```diff
@@ -0,0 +1,259 @@
+import static uk.gov.hmcts.futurehearings.hmi.unit.testing.util.TestUtilities.readFileContents;
+
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Value;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.futurehearings.hmi.Application;
+
+import uk.gov.hmcts.futurehearings.hmi.unit.testing.util.TestReporter;
+import uk.gov.hmcts.futurehearings.hmi.unit.testing.util.TestUtilities;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+@SpringBootTest(classes = {Application.class})
+@ActiveProfiles("test")
+@ExtendWith(TestReporter.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@DisplayName("PUT /resources/location - Update Location Resource")
+public class DELETE_Resources_LinkedHearingGrp_UnitTests {
+    static final String CORRECT_DELETE_LOCATION_RESOURCE_PAYLOAD = "requests/delete-resources-location-payload.json";
+
+    @Value("${targetInstance}")
+    private String targetInstance;
+
+    @Value("${targetSubscriptionKey}")
+    private String targetSubscriptionKey;
+
+    @Value("${genericResourcesApiRootContext}")
+    private String genericResourcesApiRootContext;
+
+    @Value("${destinationSystem}")
+    private String destinationSystem;
+
+    private final Map<String, Object> headersAsMap = new HashMap<>();
+
+    @Value("${tokenURL}")
+    private String tokenURL;
+
+    @Value("${clientID}")
+    private String clientID;
+
+    @Value("${clientSecret}")
+    private String clientSecret;
+
+    @Value("${scope}")
+    private String scope;
+
+    @Value("${grantType}")
+    private String grantType;
+
+    private static String accessToken;
+
+    @Value("${invalidTokenURL}")
+    private String invalidTokenURL;
+
+    @Value("${invalidScope}")
+    private String invalidScope;
+
+    @Value("${invalidClientID}")
+    private String invalidClientID;
+
+    @Value("${invalidClientSecret}")
+    private String invalidClientSecret;
+
+    private String linkedHearingGroupCtx;
+
+    @BeforeAll
+    void setToken(){
+        accessToken = TestUtilities.getToken(grantType, clientID, clientSecret, tokenURL, scope);
+        linkedHearingGroupCtx = String.format(genericResourcesApiRootContext, "linked-hearing-group/groupClientReference");
```

Comment:
Should this be a variable?

Reply:
genericResourcesApiRootContext is already a variable. "linked-hearing-group/groupClientReference" is a suffix, which may vary for other tests.